tox.ini

```diff
@@ -4,37 +4,19 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26-pytest26-supported-xdist,
-          py26-pytest27-supported-xdist,
-          py26-pytest28-supported-xdist,
-          py27-pytest26-supported-xdist,
-          py27-pytest27-supported-xdist,
-          py27-pytest28-supported-xdist,
-          py27-pytest26-unsupported-xdist,
-          py27-pytest27-unsupported-xdist,
-          py27-pytest28-unsupported-xdist,
-          py33-pytest26-supported-xdist,
-          py33-pytest27-supported-xdist,
-          py33-pytest28-supported-xdist,
-          py34-pytest26-supported-xdist,
-          py34-pytest27-supported-xdist,
-          py34-pytest28-supported-xdist,
-          py35-pytest27-supported-xdist,
-          py35-pytest28-supported-xdist,
-          py35-pytest27-unsupported-xdist,
-          py35-pytest28-unsupported-xdist,
-          pypy-pytest26-supported-xdist,
-          pypy-pytest27-supported-xdist,
-          pypy-pytest28-supported-xdist
+envlist = py27-pytest29-supported-xdist,
```

Comment:
I have to keep travis.yml and tox.ini in sync so using fancy expansions won't help here